android/BOINC/app/build.gradle

```diff
@@ -71,5 +71,5 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:26.0.2'
+    implementation 'com.android.support:appcompat-v7:26.1.0'
```

Comment:
@adamradocz, in #2896 you have the same change bu to higher version. I suppose this one can be just closed without merging

Reply:
Yes, that issue closes this one. I created two different issues because this related to Android 8, while the other one bumps the version of Android to 9 too, so a little bigger change.